.eslintrc.json

```diff
@@ -57,9 +57,11 @@
         ]
       }
     ],
-    "no-console": "off"
+    "no-console": "off",
+    "import/extensions": ["off"]
```

Comment:
@AntonOstman, I fixed eslint by disabling import/extensions check. I worked locally when the js file created by Emscripten was present, but not in GitHub when it was not. This check is not that important.

Reply:
nice!